src/mjolnir/shortcutbuilder.cc

```diff
@@ -423,9 +423,11 @@ uint32_t AddShortcutEdges(GraphReader& reader,
         std::reverse(shape.begin(), shape.end());
       }
 
-      // Get names and types - they apply over all edges of the shortcut
+      // Get names - they apply over all edges of the shortcut
       auto names = edgeinfo.GetNames();
```

Comment:
This is where we get the vectors for the names, pronunciations, and taggedvalues.  May change this for pronunciations when we work on languages.